frame/evm/src/lib.rs

```diff
@@ -361,6 +364,7 @@ decl_module! {
 
 			let (values, logs) = executor.deconstruct();
 			backend.apply(values, logs, true);
+			Module::<T>::deposit_event(Event::Created(create_address));
```

Comment:
When `res` is an error, didn't the creation failed and we should not send the event in that case?